lib/board.rb

```diff
@@ -1,9 +1,10 @@
 class Board
   attr_reader :grid
-  attr_accessor :agent
+  attr_accessor :current_player
 
   def initialize(rows, cols)
     @grid = Array.new(rows) { Array.new(cols) { Cell.new } }
+    @current_player = :player
```

Comment:
This is a hardcoded gameplay decision — player starts first. We can adjust to be anything!